lib/probe_dock_ruby/annotation.rb

```diff
@@ -60,7 +60,7 @@ def parse(str)
     end
 
     def keyword_regexp(keyword)
-      /#{keyword}=(?:(?<#{keyword}>[^"' ]+)|["']?(?<#{keyword}>[^"']+)["']?)/
+      /#{keyword}=(?:(?<#{keyword}>[^"' (?:\n|\r\n)]+)|["']?(?<#{keyword}>[^"']+)["']?)/
```

Comment:
See if it works with `[^"'\s]` instead of `[^"' (?:\n|\r\n)]`. That would be a bit less convoluted.